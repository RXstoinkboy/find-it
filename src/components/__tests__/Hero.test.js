import React from 'react'
import {Hero} from '../Hero'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'

describe('Hero component', ()=>{
    it('renders correctly',()=>{
        const wrapper = shallow(<Hero />);

        expect(toJson(wrapper)).toMatchSnapshot();
    })
})