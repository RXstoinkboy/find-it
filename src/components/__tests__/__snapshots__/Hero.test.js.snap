// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero component renders correctly 1`] = `
<section
  className="slider"
>
  <div
    className="imageWrapper"
  >
    <img
      alt="apartment info - change it to something take from API"
      className="img"
      src="logo.jpg"
    />
  </div>
  <button
    className="leftArrow"
  >
    <span
      className="leftArrow__innerEl"
    />
  </button>
  <button
    className="rightArrow"
  >
    <span
      className="rightArrow__innerEl"
    />
  </button>
  <div
    className="dots"
  >
    <div
      className="dotMarked"
    />
    <div
      className="dot"
    />
    <div
      className="dot"
    />
    <div
      className="dot"
    />
    <div
      className="dot"
    />
  </div>
</section>
`;
